source/input.h

```diff
@@ -273,15 +273,13 @@ class Input
 	double md_qmass;                   //mass of thermostat
 	double md_tfirst;                    //temperature begin
 	double md_tlast;                    //temperature end
-	int md_dumpmdfred;                  //The period to dump MD information for monitoring and restarting MD
+	int md_dumpfred;                  //The period to dump MD information for monitoring and restarting MD
```

Comment:
Why these variables are commented out?

Reply:
These variables are useless in module_MD now.